src/sub/tests/unit/test_subhub.py

```diff
@@ -178,25 +187,31 @@ def test_subscribe_success(
     """
 
     client = app.app.test_client()
-    fh = open("src/sub/tests/unit/fixtures/stripe_cust_test1.json")
+    fh = open(
+        os.path.join(os.path.dirname(__file__), "fixtures/stripe_cust_test1.json")
+    )
     cust_test1 = json.loads(fh.read())
     fh.close()
 
     mock_new_customer.return_value = convert_to_stripe_object(cust_test1)
     mock_has_plan.return_value = False
     mock_fetch_customer.return_value = convert_to_stripe_object(cust_test1)
 
-    fh = open("src/sub/tests/unit/fixtures/stripe_sub_test1.json")
+    fh = open(os.path.join(os.path.dirname(__file__), "fixtures/stripe_sub_test1.json"))
     sub_test1 = json.loads(fh.read())
     fh.close()
-    fh = open("src/sub/tests/unit/fixtures/stripe_plan_test1.json")
+    fh = open(
+        os.path.join(os.path.dirname(__file__), "fixtures/stripe_plan_test1.json")
```

Comment:
also consider calculating the dirname of __file__ once and then using it; such that you lines aren't so long that black does this ugly wrapping

Reply:
Done (good suggestion)